packages/accounts/package.json

```diff
@@ -29,6 +29,8 @@
     "./package.json": "./package.json"
   },
   "scripts": {
+    "generate": "npx wagmi generate",
```

Comment:
learn: have you worked with the wagmi cli? if not / maybe if you remember when you first did, did you just look up a way to generate abis and some stackoverflow pointed you to it? 

otherwise, curious how you arrived at this solution.

Reply:
I used it before, but then looked at their codebase for how they define plugins so I could write my own